### PR TITLE
UI ux 81 clean up code

### PR DESCRIPTION
**Pull request by jayjayjayjayjayjay**

### CODE REVIEW

**Comment by jayjayjayjayjayjay** [comment]
Removed a fair amount of unused code.

File: frontend_new/src/components/Search.tsx [hunk 1]
```diff
@@ -109,21 +108,20 @@ function SearchInfo(props: props) {
...
+          color="primary"
           fullWidth
           onClick={() => getRes(searchInput, true)}
         >
           Associative Search
         </Button>
       </Grid>
       <InfoButton text="This is a description" />
-      <Grid item xs={12} style={{ display: props.isVisible && "none" }}>
+      <Grid item xs={12} style={{ display: String(props.isVisible) }}>
```
Comment (ariadev79) [inline frontend_new/src/components/Search.tsx:119]: **[Question]** Would just `props.isVisible` not work?

File: frontend_new/src/components/ThemeSwitch.tsx [hunk 2]
```diff
@@ -50,17 +50,16 @@ const MaterialUISwitch = styled(Switch)(({ theme }) => ({
...
-  whichtheme: theme;
+  themeChange: () => void;
 };
 
 export default function CustomizedSwitches(props: props) {
   const theme = useTheme();
   return (
     <MaterialUISwitch
-      sx={{ m: 1 }}
+      //@ts-ignore:
```
Comment (ariadev79) [inline frontend_new/src/components/ThemeSwitch.tsx:60]: **[Question]** Out of curiosity, what's this about?

File: frontend_new/src/components/Search.tsx (hunk 1 above)
Comment (jayjayjayjayjayjay) [reply frontend_new/src/components/Search.tsx:119]: typescript was throwing an issue and this was my solution. As isVisible is a boolean, but we want is as a string so casting to a string

File: frontend_new/src/components/ThemeSwitch.tsx (hunk 2 above)
Comment (jayjayjayjayjayjay) [reply frontend_new/src/components/ThemeSwitch.tsx:60]: get typescript to not check line below

**Review by ariadev79 — APPROVED** [review]

**Pull request merged**